cluster-scope/overlays/prod/emea/morty/ingresscontrollers/default.yaml

```diff
@@ -4,13 +4,7 @@ metadata:
   name: default
   namespace: openshift-ingress-operator
 spec:
-  clientTLS:
-    clientCA:
-      name: ""
-    clientCertificatePolicy: ""
   httpEmptyRequestsPolicy: Respond
-  httpErrorCodePages:
-    name: ""
```

Comment:
These are valid options in OCP 4.9, removing them will change the cluster behavior.

https://docs.openshift.com/container-platform/4.9/networking/ingress-operator.html

Reply:
Resolved in latest commit